src/common/dto/courses/CreateCourse.dto.ts

```diff
@@ -33,56 +32,51 @@ export abstract class CreateCourse {
   })
   @IsString()
   @IsNotEmpty()
+  @Transform(trimIfString)
   public title: string;
 
   @ApiModelProperty({
     type: 'string',
     example: 'CS',
   })
   @IsString()
-  @IsNotEmpty()
-  public prefix: string;
+  @IsOptional()
+  @Transform(trimIfString)
+  public prefix?: string;
 
   @ApiModelProperty({
     type: 'string',
     example: '109b',
   })
   @IsString()
-  @IsNotEmpty()
-  public number: string;
+  @IsOptional()
+  @Transform(trimIfString)
+  public number?: string;
 
   @ApiModelProperty({
     type: 'string',
     example: 'CS 050',
   })
   @IsString()
   @IsOptional()
+  @Transform(trimIfString)
   public sameAs?: string;
 
   @ApiModelProperty({
     type: 'string',
     enum: IS_SEAS,
-    example: true,
+    example: IS_SEAS.Y,
   })
   @IsEnum(IS_SEAS)
   @IsNotEmpty()
   public isSEAS: IS_SEAS;
 
-  @ApiModelProperty({
-    type: 'string',
-    example: 'Taking place in a larger room this year',
-    default: '',
-  })
-  @IsString()
-  @IsOptional()
-  public notes?: string;
-
   @ApiModelProperty({
     type: 'boolean',
     example: false,
   })
-  @IsNotEmpty()
   @IsBoolean()
+  @IsOptional()
```

Comment:
If this is optional - what's the default value of it?

Reply:
I thought that the default was being set to `true` by this code in the course entity? Should I not have the `@IsOptional` decorator?

Reply:
Sure - I was more meaning that you might want to add the default value in to the `@ApiModelProperty` decorator for the documentation :)

Reply:
Ah, I see, ok, thanks! I updated the `@ApiModelProperty` for the create and update course DTOs in 905b8f8.